src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java

```diff
@@ -1067,9 +1067,83 @@ public void testRegisteringOutputByIndexandAcquiringOutputParamByName() throws S
         }
     }
 
+    @Test
+    public void testExecuteSystemStoredProcedureNamedParametersAndIndexedParameterNoResultset() throws SQLException {
+        String call = "EXEC sp_getapplock @Resource=?, @LockTimeout='60', @LockMode='Exclusive', @LockOwner='Session'";
```

Comment:
can we choose a shorter timeout value? in case of problems then this won't have to wait too long